ios/browser/brave_wallet/eth_tx_service_factory.h

```diff
@@ -27,6 +27,7 @@ class EthTxServiceFactory : public BrowserStateKeyedServiceFactory {
   // Creates the service if it doesn't exist already for |browser_state|.
   static mojo::PendingRemote<mojom::EthTxService> GetForBrowserState(
       ChromeBrowserState* browser_state);
+  static KeyringService* GetServiceForState(ChromeBrowserState* browser_state);
```

Comment:
Looks like this should be `EthTxService`

Reply:
Ya... fixed :)
Relying on CI to catch my dumb mistakes for iOS